src/collections/components/CollectionForm/CollectionForm.tsx

```diff
@@ -108,7 +108,8 @@ export const CollectionForm: React.FC<
     // before they actually submit the form
     validateOnBlur: false,
     validateOnChange: false,
-    validationSchema: getValidationSchema(authorIds, languages),
+    validationSchema: getValidationSchema(authorIds),
+
```

Comment:
I have a feeling these changes do not belong here.

Reply:
Yes these are from the merge as well. However, I realized that after Jonathan's PR the `languages` argument was not being used in that validation function so I went ahead and removed it.

I've caused confusion so I'll go back and remove it 😅